src/js/serial.js

```diff
@@ -187,6 +189,28 @@ const serial = {
             }
         });
     },
+    connectVirtual: function (callback) {
+        var self = this;
+        self.openRequested = true;
```

Comment:
```suggestion
        self.openRequested = true;
```
Probably can remove `openRequested` as I removed it in latest changes because it seems not used in any logic